libcaf_net/test/multiplexer.cpp

```diff
@@ -70,7 +73,7 @@ class dummy_manager : public socket_manager {
   bool handle_write_event() override {
     if (wr_buf_.size() == 0)
       return false;
-    auto res = write(handle(), wr_buf_.data(), wr_buf_.size());
+    auto res = write(handle(), as_bytes(make_span(wr_buf_)));
```

Comment:
Should also work without `as_bytes`.

```suggestion
    auto res = write(handle(), make_span(wr_buf_));
```

Reply:
Same thing as above.